src/objective-c/tests/Podfile

```diff
@@ -92,6 +92,10 @@ post_install do |installer|
   installer.pods_project.targets.each do |target|
     target.build_configurations.each do |config|
       config.build_settings['GCC_TREAT_WARNINGS_AS_ERRORS'] = 'YES'
+      if config.name.include?("Test")
+        config.build_settings['GCC_OPTIMIZATION_LEVEL'] = '0'
```

Comment:
Also just curious if there is any chance that certain bugs only manifest in optimized mode?

Reply:
that's good point, actually. Ideally we don't have to (otherwise that would be very tricky issue to fix), but I think there is value to run our test suite close to a release config;  Updated here to override the command line run script to use original config (-Os) so our kokoro test run still is optimized build.